### DISCUSSION TITLE
Should we switch to Pingnoo.2021.06.22-develop.universal.dmg?

### DISCUSSION THREAD

**Discussion by chmac** [Homebrew/discussions#5436] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

The release marked as "stable" on GitHub is an x86 only build. There is a later release that's a universal binary, but it's not been marked as stable. The project hasn't been updated since 2021 so it seems like the repo won't be updated anytime soon. But the universal binary does work today.

My question is, should I submit a PR to switch to the universal binary? I'm not familiar with brew policies around this.

**Comment by SMillerDev** [comment]
I'd suggest making an upstream issue asking if it's stable.

**Reply by chmac** [reply]
Thanks for the quick response! The upstream repo has been abandoned since 2021 it seems like. There's no recent replies from the author. So I could open such an issue, but I don't think I'll get any kind of response from the author soon. I was recommended to install that version by another user via [this issue](https://github.com/nedrysoft/pingnoo/issues/112). Do you think that's a good enough signal that this is a stable version?

**Reply by p-linnane** [reply, marked as answer]
No. If upstream is dead, we should leave it as is and add a deprecation notice.

**Reply by chmac** [reply]
Looks like upstream might not be as dead as I imagined, according to the [last update](https://github.com/nedrysoft/pingnoo/issues/112#issuecomment-2165182187) here. Hopefully the author will update the `latest` tag then.
